docs/csharp/tour-of-csharp/index.md

```diff
@@ -5,9 +5,11 @@ ms.date: 04/05/2019
 ms.custom: seoapril2019
 ---
 
-# A Tour of the C# Language
+# A tour of the C# language
 
-C# (pronounced "See Sharp") is a simple, modern, object-oriented, and type-safe programming language. C# has its roots in the C family of languages and will be immediately familiar to C, C++, Java, and JavaScript programmers. This tour provides an overview of the major components of the language. If you want to explore the language through interactive examples, try our [introduction to C#](../tutorials/intro-to-csharp/index.md) tutorials.
+C# (pronounced "See Sharp") is a simple, modern, object-oriented, and type-safe programming language. C# has its roots in the C family of languages and will be immediately familiar to C, C++, Java, and JavaScript programmers.
+
+This tour provides an overview of the major components of the language in C# 7 and later. If you want to explore the language through interactive examples, try the [introduction to C#](../tutorials/intro-to-csharp/index.md) tutorials.
```

Comment:
Maybe, the first sentence should be rephrased. As it is now, I would expect, for example, the nullable reference types feature to be included into this tour, because it's the major component of C# 8.0. 

It looks that those articles were made based on the draft C# 6 specification. Then the following sentence might be an alternative:

> This tour provides an overview of the major components of C# 6.

The only post-C# 6 feature that I've noticed in the tour is the [updated list of access modifiers](https://docs.microsoft.com/en-us/dotnet/csharp/tour-of-csharp/classes-and-objects#accessibility) (`private protected` was added in C# 7.2). However, nothing about pattern matching and value tuples that are available beginning with C# 7.0.

Reply:
@pkulikov You're correct that this is outdated. Unfortunately, it's outdated even more than 6.0.

The history is that this section was moved out of the C# ECMA standard as part of the ECMA 5.0 work. It's up to date as of C# 2.0 😮 I have a planned task to update this entire section. That will address [a number of issues](https://github.com/dotnet/docs/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+%22tour-of-csharp%22+). Rather than expand the scope of this PR, I'd like to merge this, and address the other issues in one update PR.